### PR TITLE
Added missing keys to render MilestoneItem

### PR DESCRIPTION
**Pull request by ascariandrea**
#336

Maybe I'm wrong, but I think there's a missing `etherAmount` key in the response received from the API.

Can someone please confirm this?

Or tell me I'm stupid is also an option :)

### CODE REVIEW

**Comment by vojtechsimetka** [comment]
Deploy preview for *giveth-dapp* processing.

Built with commit 8f380962a6cc4a16bc66eea3d0ec5ff92ce82eb6

https://app.netlify.com/sites/giveth-dapp/deploys/5ae8a0428df8947532c12dda

**Comment by ascariandrea** [comment]
I reverted my changes and also drop the `develop` branch merging commit.

But my question is still unanswered :/

Should the `item` have `etherAmount` key?

**Comment by vojtechsimetka** [comment]
I don't have time to properly check now, but I think so. Look at the Rate Convertor component (which should be actually used in the milestone edit itself because we are just repeating the logic).

File: src/components/views/ViewMilestone.jsx [hunk 1]
```diff
@@ -255,7 +254,19 @@ class ViewMilestone extends Component {
...
-                                <MilestoneItem name={`milestoneItem-${i}`} item={item} />
+                                <MilestoneItem
+                                  key={item.date}
+                                  name={`milestoneItem-${i}`}
+                                  item={{
+                                    date: item.date,
+                                    description: item.description,
+                                    selectedFiatType,
+                                    fiatAmount,
+                                    conversionRate,
```
Comment (vojtechsimetka) [inline src/components/views/ViewMilestone.jsx]: The selectedFiatType, fiatAmount and conversionRate are surely (and if not should be) defined and stored per item.

**Comment by vojtechsimetka** [comment]
So the way milestones work now is:

- Milestones can from now on be denominated in one token or ETH.
- Each item can however have a conversion from any fiat currency or ETH.

**Comment by ascariandrea** [comment]
I fixed using the item's values for `MilestoneItem` rendering

**Pull request merged**